cfme/services/catalogs/catalog.py

```diff
@@ -126,56 +115,38 @@ def exists(self):
 class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
-    def am_i_here(self):
-        return match_page(summary='All Catalogs')
+    VIEW = CatalogsView
 
     def step(self):
         self.prerequisite_view.navigation.select('Services', 'Catalogs')
-        tree = accordion.tree('Catalogs')
-        tree.click_path('All Catalogs')
-
-    def resetter(self):
-        # Default list view
-        tb.select('List View')
-        if paginator.page_controls_exist():
-            # Make sure nothing is selected
-            sel.check(paginator.check_all())
-            sel.uncheck(paginator.check_all())
+        self.view.catalogs.tree.click_path("All Catalogs")
 
 
-@navigator.register(Catalog, 'Add')
-class Add(CFMENavigateStep):
+@navigator.register(Catalog, 'AddCatalog')
```

Comment:
Why the change? Convention says that we should use Add/Edit/Details and such names, so you never have to look it up. If you have multiple classes defined in one module and the Add steps' classes' names clash, you can rename the class but keep the step name as Add ...